src/main/java/de/srendi/advancedperipherals/common/addons/APAddons.java

```diff
@@ -1,12 +1,7 @@
 package de.srendi.advancedperipherals.common.addons;
 
-import de.srendi.advancedperipherals.AdvancedPeripherals;
-import net.neoforged.bus.api.SubscribeEvent;
 import net.neoforged.fml.ModList;
-import net.neoforged.fml.common.EventBusSubscriber;
-import net.neoforged.fml.event.lifecycle.InterModEnqueueEvent;
 
-@EventBusSubscriber(modid = AdvancedPeripherals.MOD_ID, bus = EventBusSubscriber.Bus.MOD)
 public class APAddons {
```

Comment:
maybe we can just make this as enum instead?

Reply:
I made an enum out of it, how I think it would be the best
What are your opinions of the approach I took?